app/helpers/application_helper.rb

```diff
@@ -1,6 +1,6 @@
 module ApplicationHelper
 
-  def full_title(page_title = '')
+def full_title(page_title = '')
```

Comment:
- lùi vào đầu dòng 2 spaces nhé em
- Xóa ()
- Dùng " thay cho ' trong toàn bộ file này